### PR TITLE
Reorganizing and commenting terraform.tfvars for easier end-user modifications

### PR DESCRIPTION
**Pull request by VimCommando**
Also some minor house keeping on the `.gitignore` file to filter out generated log files and removing excess line breaks in the `variables.tf` files.

### CODE REVIEW

**Comment by VimCommando** [comment]
Not ready to merge this yet, the updated tagging conventions break some Terraform scripts.

File: aws/terraform.tfvars [hunk 1]
```diff
@@ -1,158 +1,176 @@
...
+#======================= Deployment Configuration ==============================
+
+# House keeping tags. For internal Elastic projects, please follow the cloud
+# resource tagging guidelines.
 
-##house keeping tags.  Set please
 tags = {
-    "Owner" = "sunile.manjee@.co"
-    "KeepAliveUntil" = "12312023"
-    "username" = "sunman" #This can be anything you call yourself
```
Comment (sunilemanjee) [inline aws/terraform.tfvars]: removing username will cause an issue with the automation.  I'll need to swap username with project for this to work.  Not a big issue but needs to be tested.

File: .gitignore [hunk 2]
```diff
@@ -103,7 +102,6 @@ gcp/create-gke/addons/openebs/terraform.tfstate.backup
 gcp/create-gke/addons/openebs/tflogs/
```
Comment (sunilemanjee) [inline .gitignore]: what are you thoughts about doing this instead?
`gcp/*tflogs
gcp/*/*tflogs
gcp/*/*/*tflogs`

**Review by sunilemanjee — COMMENTED** [review]
Reviewed.  My biggest concern is removing username from tags.  Allow me to change to project within the terraform resources and test.

File: .gitignore (hunk 2 above)
Comment (VimCommando) [reply .gitignore]: That makes sense, since you have multiple sub-directories with other log files too.

Comment (VimCommando) [reply .gitignore]: I think we can do one better:
```
*.log
**/*logs
```

This will match all `*.log` files and any `*logs` directories.

**Review by sunilemanjee — APPROVED** [review]
Thanks for your contributions!  Variable organization looks great plus the bug fix.

**Pull request merged**